staging/src/k8s.io/component-base/flagz/flagz_test.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flagz
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/spf13/pflag"
+	cliflag "k8s.io/component-base/cli/flag"
+)
+
+func TestFlags(t *testing.T) {
+	fs1 := pflag.NewFlagSet("test1-set", pflag.ContinueOnError)
+	flagValue1 := ""
+	fs1.StringVar(&flagValue1, "test1-flag", "test1-value", "test1-usage")
+
+	fs2 := pflag.NewFlagSet("test2-set", pflag.ContinueOnError)
+	flagValue2 := ""
+	fs2.StringVar(&flagValue2, "test2-flag", "test2-value", "test2-usage")
+
```

Comment:
we should also add a test for sensitive flags

Reply:
Ah yes, good point. Added